plugins/org/src/components/Cards/OwnershipCard/useGetEntities.ts

```diff
@@ -57,78 +58,122 @@ const getQueryParams = (
   return qs.stringify({ filters }, { arrayFormat: 'repeat' });
 };
 
-const getOwnersEntityRef = (owner: Entity): string[] => {
-  let owners = [stringifyEntityRef(owner)];
-  if (owner.kind === 'User') {
-    const ownerGroups = getEntityRelations(owner, RELATION_MEMBER_OF, {
-      kind: 'Group',
-    });
-    const ownerGroupsName = ownerGroups.map(ownerGroup =>
-      stringifyEntityRef({
-        kind: ownerGroup.kind,
-        namespace: ownerGroup.namespace,
-        name: ownerGroup.name,
-      }),
+const getMemberOfEntityRefs = (owner: Entity): string[] => {
+  const parentGroups = getEntityRelations(owner, RELATION_MEMBER_OF, {
+    kind: 'Group',
+  });
+
+  const ownerGroupsNames = parentGroups.map(({ kind, namespace, name }) =>
+    stringifyEntityRef({
+      kind,
+      namespace,
+      name,
+    }),
+  );
+
+  const {
+    kind,
+    metadata: { namespace, name },
+  } = owner;
+
+  return [...ownerGroupsNames, stringifyEntityRef({ kind, namespace, name })];
```

Comment:
```suggestion
  return [...ownerGroupsNames, stringifyEntityRef(owner)];
```

no need to unpack these; the function accepts an entity directly since this is such a common thing to want to do

Reply:
Good point, must have missed that signature :+1: